src/components/SignIn/SignIn.js

```diff
@@ -3,14 +3,14 @@ import React from 'react';
 import PopupWithForm from '../PopupWithForm/PopupWithForm';
```

Comment:
`PopupWithForm ` should contain the form and the submit button inside it. It's the main purpose of `PopupWithForm `. Please, make it the same way you had in the 14th sprint